File: src/main/java/com/project/mgz/configs/MvcConfig.java
package com.project.mgz.configs;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

@Configuration
public class MvcConfig extends WebMvcConfigurerAdapter {
    private static final String HOME_VIEW = "home";
    private static final String LOGIN_VIEW = "login";
    private static final String HELLO_VIEW = "hello";

    @Override
    public void addViewControllers(ViewControllerRegistry registry) {
        registry.addViewController("/home").setViewName(HOME_VIEW);
        registry.addViewController("/").setViewName(HOME_VIEW);
        registry.addViewController("/login").setViewName(LOGIN_VIEW);
        registry.addViewController("/hello").setViewName(HELLO_VIEW);
    }
}

File: serveit/src/main/kotlin/com/project/serveit/config/MvcConfig.kt
package com.project.serveit.config

import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class MvcConfig : WebMvcConfigurer {
    override fun addViewControllers(registry: ViewControllerRegistry) {
        registry.run {
            addViewController("/home").setViewName("home")
            addViewController("/").setViewName("home")
            addViewController("/login").setViewName("login")
            addViewController("/hello").setViewName("hello")
        }
    }

}
